packages/scripts/package.json

```diff
@@ -30,10 +32,16 @@
   },
   "scripts": {
     "test": "mocha 'test/**/*.js'",
+    "prebuild": "ts-node generate-types.ts",
+    "clean": "rm -rf build",
     "build": "tsc --build"
   },
   "publishConfig": {
     "registry": "https://registry.npmjs.com/",
     "access": "public"
+  },
+  "devDependencies": {
+    "ts-node": "^10.9.2",
+    "typescript": "^5.8.3"
```

Comment:
`@safe-global/types-kit` was not compatible with older typescipt from the root of the workspace.